src/test/java/org/isf/generaldata/GeneralDataTest.java

```diff
@@ -0,0 +1,22 @@
+package org.isf.generaldata;
+
+import static org.junit.Assert.*;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class GeneralDataTest {
+
+	@Ignore
+	@Test
+	public void testGetGeneralData() {
+		GeneralData generalData =  GeneralData.getGeneralData();
+		
+		assertNotNull(generalData);
```

Comment:
In Core we've switched to assertThat() style, so: `assertThat(generalData).isNotNull();`

Reply:
Fixed